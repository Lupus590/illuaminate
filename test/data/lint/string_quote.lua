-- config: (lint (only format:string-quote))

print("foo", 'foo', [[foo]])
print("'\n\"\'", '"\n\"\'', [[
"']])
