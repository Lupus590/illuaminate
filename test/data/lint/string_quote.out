string_quote.lua:[3:14-3:18]: String should use double quotes ('"') [format:string-quote]
   │
 3 │ print("foo", 'foo', [[foo]])
   │              ^^^^^
string_quote.lua:[4:18-4:26]: String should use double quotes ('"') [format:string-quote]
   │
 4 │ print("'\n\"\'", '"\n\"\'', [[
   │                  ^^^^^^^^^
No errors and 2 warnings
 -- config: (lint (only format:string-quote))
 
-print("foo", 'foo', [[foo]])
-print("'\n\"\'", '"\n\"\'', [[
+print("foo", "foo", [[foo]])
+print("'\n\"\'", "\"\n\"\'", [[
 "']])
 
